spec/requests/note_request_spec.rb

```diff
@@ -0,0 +1,260 @@
+require 'rails_helper'
+
+RSpec.describe Note, type: :request do
+
+  let(:note) { create(:note) }
+  let(:show_path) { note_path(note.friendly_url) }
+  let(:person) { create(:person) }
```

Comment:
I am fairly new to all of RSpec and I hadn't seen `let` used before these PRs - I really like the way you're using it!